documentation/src/content/docs/en/essentials/manage-states.mdx

```diff
@@ -0,0 +1,240 @@
+:::
+
+## Reading Values (#read-store-value)
+
+There are several ways to get the current value of a store:
+
+1. getState() method
+
+```ts
+console.log($counter.getState()); // 0
+```
+
+2. Subscribe to changes via watch
+
+```ts
+$counter.watch((value) => {
+  console.log("Counter changed:", value);
+});
+```
+
+3. Using framework integration hooks
+
+<Tabs>
+  <TabItem label="React">
+
+```ts
+import { useStore } from 'effector-react'
+import { $counter } from './model.js'
+
+const Counter = () => {
+  const counter = useUnit($counter)
+
+  return <div>{counter}</div>
+}
+```
+
+  </TabItem>
+  <TabItem label="Vue">
+
+```html
+<script setup>
+  import { useUnit } from "effector-vue/composition";
+  import { $counter } from "./model.js";
+
+  const counter = useUnit($counter);
+</script>
+```
+
+  </TabItem>
+  <TabItem label="Solid">
+
+```ts
+import { useStore } from 'effector-solid'
+import { $counter } from './model.js'
+
+const Counter = () => {
+  const counter = useUnit($counter)
+
+  return <div>{counter()}</div>
+}
+```
+
+  </TabItem>
+</Tabs>
+
+## Store Updates (#store-updates)
+
+### Updating via Events (#update-store-via-events)
+
+The simplest way to update a store is to bind it to an event:
+
+```ts
+import { createStore, createEvent } from "effector";
+
+const incremented = createEvent();
+const resetCounter = createEvent();
+
+const $counter = createStore(0)
+  // Increase value by 1 each time the event is called
+  .on(incremented, (state) => state + 1)
+  // Reset value to 0
+  .reset(resetCounter);
+
+$counter.watch((state) => console.log(state));
+
+// Usage
+incremented();
+incremented();
+
+resetCounter();
+
+// Console output
+// 0 - output on initialization
+// 1
+// 2
+// 0
+```
+
+### Updating with Event parameters (#update-stores-with-event-params)
+
+You can update a store using event parameters by passing data to the event like a regular function and using it in the handler.
+
+```ts
+import { createStore, createEvent } from "effector";
+
+const userUpdated = createEvent<{ name: string }>();
+
+const $user = createStore({ name: "Bob" });
+
+$user.on(userUpdated, (state, update) => ({
+  ...state,
+  ...update,
+}));
+
+userUpdated({ name: "Alice" });
+```
+
+### Multiple Updates
+
+A store isn't limited to a single event subscription - you can subscribe to as many events as you need, and different stores can subscribe to the same event.
+
+```ts
+const categoryChanged = createEvent<string>();
+const searchQueryChanged = createEvent<string>();
+const filtersReset = createEvent();
+
+const $lastUsedFilter = createStore<string | null>(null);
+const $filters = createStore({
+  category: "all",
+  searchQuery: "",
+});
+
+// subscribe two different stores to the same event
+$lastUsedFilter.on(categoryChanged, (_, category) => category);
+$filters.on(categoryChanged, (state, category) => ({
+  ...state,
+  category,
+}));
+
+$filters.on(searchQueryChanged, (state, searchQuery) => ({
+  ...state,
+  searchQuery,
+}));
```

Comment:
I think it is better do not recommend using names like state|data|payload without real need.

```suggestion
$filters.on(categoryChanged, (filters, category) => ({
  ...filters,
  category,
}));

$filters.on(searchQueryChanged, (filters, searchQuery) => ({
  ...filters,
  searchQuery,
}));
```

Reply:
Please, check all other documentation changes for useless arguments names like state/data/payload/etc